chart_app/lib/deriv_chart_wrapper.dart

```diff
@@ -366,7 +366,6 @@ class DerivChartWrapperState extends State<DerivChartWrapper> {
                           : configModel.msPerPx,
                       minIntervalWidth: _getMinIntervalWidth(),
                       maxIntervalWidth: _getMaxIntervalWidth(),
-                      dataFitPadding: _getDataFitPadding(rightPadding),
```

Comment:
just a question: could you please tell more about why we need to remove it? :)
If we don't need it, it might be better to remove unused  _getDataFitPadding() declaration

Reply:
Initially, I included the _getDataFitPadding() declaration because I encountered some errors locally that seemed to be resolved on it's removal . But it works fine on the testlink  as I checked now . I added it back .